docs/09-英文/01-Introduction/02quickstart.md

```diff
@@ -0,0 +1,93 @@
+# Quickstart
+
+This section will introduce the method of quickly building and deploying a single-Master RocketMQ cluster to complete simple message sending and receiving.
+
+## 1. Get Apache RocketMQ
+
+:::tip Download RocketMQ
```

Comment:
`:::tip` (as well as `:::` in other place) is still displayed as original text, is it expected?

Reply:
Hey @caigy , these are the syntax for the frontend.